src/components/NewClusterModal.js

```diff
@@ -122,6 +122,7 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
   getRuntimeConfig() {
     return formatRuntimeConfig({
       cloudService: !!this.state.sparkMode ? 'DATAPROC' : 'GCE',
+      bootDiskSize: !!this.state.sparkMode ? null : 50,
```

Comment:
TOL: This assumption will work for newly created runtimes after the Leo change, but may slightly over-estimate costs for updates to old clusters. That may be fine, especially considering it's such a small amount and only affects this one particular case, (plus slightly over-estimating is better than under-estimating).

Reply:
Good point. To account for that, we'd need to:
* know that there is an existing runtime and whether or not it has a boot disk
* know whether or not the change being made is going to result in deleting and recreating the runtime
This is a lot of context that this function currently doesn't have.

Also, darn, I just noticed that this is used to generate the payload to leo where it's technically incorrect and possibly misleading to be providing a `bootDiskSize` property. Unless leo actually does accept this property... but I'm not sure we want to go down that road anyway.

Reply:
I thought about this a little more. `NewClusterModal.hasChanges()` uses `_.isEqual()` on 2 configs that are first run through `normalizeRuntimeConfig()`. Maybe `normalizeRuntimeConfig()` is the right place for this so that we have better control over what constitutes a difference between a config we build to use as a `POST` payload and a config we get back from leo. `runtimeConfigCost()` already calls `normalizeRuntimeConfig()` so that should work. I haven't yet studied other callers of `normalizeRuntimeConfig()` to look for unintended consequences.